tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -7,21 +6,30 @@ import Selector, {SelectedIcon} from '../components/Selector';
 import Icon from 'react-native-vector-icons/MaterialIcons';
 import { GETYearData } from '../../services/CalendarService';
 import { VIEWS } from '../../services/utils/constants';
-import {getISODate, initializeEmptyYear} from '../../services/utils/helpers';
+import { getISODate, getMonthsDiff, initializeEmptyYear } from '../../services/utils/helpers';
 import { useFocusEffect } from '@react-navigation/native';
+import { GETJoinedDate } from '../../services/OnboardingService';
 import ErrorFallback from "../../error/error-boundary";
 import { CALENDAR_STACK_SCREENS } from '../CalendarNavigator';
 import LoadingVisual from '../components/LoadingVisual';
 import { GETTutorial } from '../../services/TutorialService';
 import LegendButton from "../../../ios/tppapp/Images.xcassets/icons/legend_icon.svg";
 
+export let scrollDate = getISODate(new Date());
 
-export const Calendar = ({navigation, marked, setYearInView, selectedView}) => {
+var joinedDate = ""; 
+GETJoinedDate().then(res => { joinedDate = res })
+const pastScroll = 12 + (getMonthsDiff(joinedDate))
```

Comment:
can you move this inside CalendarScreen maybe in a useEffect()? And prefer using ```let``` instead of ```var```